### PR TITLE
Replace all MathHelper.Clamp usages with Math.Clamp

### PR DESCRIPTION
**Pull request by turbedi**
.NET Standard 2.1 implements Math.Clamp , use it instead of MathHelper.Clamp from osuTK.

During quick testing I noticed that now an exception is thrown during the DelayedLoadWrapper visual test. 
[Exception](https://pastebin.com/14a2abPn)
I've added a commit that fixes the test, but I'm not completely sure if this is the correct fix.

### CODE REVIEW

File: osu.Framework/Graphics/Containers/ScrollContainer.cs [hunk 1]
```diff
@@ -474,7 +474,7 @@ private void updatePosition()
             // then we should handle the clamping force. Note, that if the target is _within_
             // acceptable bounds, then we do not need special handling of the clamping force, as
             // we will naturally scroll back into acceptable bounds.
-            if (!IsDragging && Current != Clamp(Current) && target != Clamp(target, -0.01f))
+            if (!IsDragging && Current != Clamp(Current) && target != Clamp(target, 0.01f))
```
Comment (Tom94) [inline osu.Framework/Graphics/Containers/ScrollContainer.cs]: Pretty confident this is not the correct fix. Probably best to check what the actual behavior was beforehand (i.e., how `MathHelper.Clamp` handled min>max) and manually implementing that behavior in `ScrollContainer.Clamp`.

Comment (peppy) [reply osu.Framework/Graphics/Containers/ScrollContainer.cs]: yep, needs proper testing and understanding before this is merged.

Comment (turbedi) [reply osu.Framework/Graphics/Containers/ScrollContainer.cs]: from osuTK:
`        public static float Clamp(float n, float min, float max)
        {
            return Math.Max(Math.Min(n, max), min);
        }`
So when min>max, then min is returned in every case.
Should I change it to how osuTK implemented it?

Comment (peppy) [reply osu.Framework/Graphics/Containers/ScrollContainer.cs]: @Tom94 i'm having trouble understanding where the -0.01f comes from in the first place (and its purpose). the comment doesn't seem to touch on it and removing it has no discernible visual change.

Comment (Tom94) [reply osu.Framework/Graphics/Containers/ScrollContainer.cs]: On quick inspection: removing the -0.01 seems to disable the strong clamping force (setting `localDistanceDecay` within the if-block) once `target` approached its clamped position. Pretty sure that'd be unintentional.

**Comment by turbedi** [comment]
Reverted the behavior of ScrollContainer.Clamp, it now uses the old Clamp implementation from osuTK again.

File: osu.Framework/Graphics/Containers/ScrollContainer.cs [hunk 2]
```diff
@@ -474,7 +474,7 @@ private void updatePosition()
             // then we should handle the clamping force. Note, that if the target is _within_
             // acceptable bounds, then we do not need special handling of the clamping force, as
             // we will naturally scroll back into acceptable bounds.
-            if (!IsDragging && Current != Clamp(Current) && target != Clamp(target, 0.01f))
+            if (!IsDragging && Current != Clamp(Current) && target != Clamp(target, -0.01f))
```
Comment (swoolcock) [inline osu.Framework/Graphics/Containers/ScrollContainer.cs]: You should not be passing a negative value here as it makes no sense.  Fix `Clamp` such that it doesn't require a negative.

Comment (turbedi) [reply osu.Framework/Graphics/Containers/ScrollContainer.cs]: I have just changed the value back to how it was before and I can't wrap my head around how to fix it correctly without breaking the strong clamp force

**Pull request merged**